docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   etl-prod:
     container_name: etl-rest-server-prod
-    image: ampathke/etl-services:v2.16.5.2
+    image: ampathke/etl-services:POC-399
```

Comment:
Does this mean, we will be changing this in every PR??

Reply:
Ideally, it's supposed to be the docker image name for the deployed commit